src/Row.ts

```diff
@@ -37,15 +37,44 @@ class Row extends RowBase<RowProperties> {
 				classes: this.classes(tableCss.table, css.rowTable)
 			}, [
 				v('tr', columns.map((column) => {
-					const { id, field } = column;
+					const { field, id } = column;
+
+					let value: any;
+					if (column.get) {
+						// Get the value from the column callback
+						value = column.get(item, column);
+					}
+					else if (!column.render) {
+						// Get the value using a property lookup on the item data
+						value = item.data[ field || id ];
+					}
+					else {
+						// This column has no value
+						value = '';
+					}
+
+					let content: DNode;
+					if (column.render) {
```

Comment:
Then do you think it would be more ergonomic to have an options style API? 
```ts
interface CustomRenderOptions {
    item: any;
    column: Column;
    value: any;
}

// consumer implements function and only uses what they need from the options
function customRender(options: CustomRenderOptions): DNode {
    // do something with either options.item, options.column or options.value
    // or any combination.
}
```